src/components/StudentGroup/AddUserPopup.tsx

```diff
@@ -37,52 +255,28 @@ const AddUserPopup = observer((props: Props) => {
-                <div className={clsx('card__header', styles.header)}>
-                    <h3>Benutzer:in hinzufügen</h3>
-                </div>
-                <div className={clsx('card__body')}>
-                    <input
-                        type="text"
-                        placeholder="Suche..."
-                        value={searchFilter}
-                        className={clsx(styles.textInput)}
-                        onChange={(e) => {
-                            setSearchFilter(e.target.value);
-                        }}
-                    />
-                    <div className={styles.listContainer}>
-                        <ul className={clsx(styles.students, styles.list)}>
-                            {userStore.users
-                                .filter((user) => searchRegex.test(user.searchTerm))
-                                .map((user, idx) => (
-                                    <li
-                                        key={idx}
-                                        className={clsx(
-                                            styles.listItem,
-                                            group.userIds.has(user.id) && styles.disabled
-                                        )}
-                                        title={user.email}
-                                    >
-                                        {user.nameShort}
-                                        <div className={styles.actions}>
-                                            <Button
-                                                onClick={() => {
-                                                    group.addStudent(user);
-                                                }}
-                                                disabled={group.userIds.has(user.id)}
-                                                icon={mdiAccountPlus}
-                                                color="green"
-                                            />
-                                        </div>
-                                    </li>
-                                ))}
-                        </ul>
-                    </div>
+                <div className={clsx(styles.addUserCardTitle)}>
+                    <Icon path={mdiAccountMultiple} size="1.4em" />
+                    <h2>{props.studentGroup.name}</h2>
                 </div>
+                <Tabs>
+                    <TabItem value="add" label="Benutzer:in hinzufügen">
+                        <AddIndividualUsersPopup {...props} popupRef={popupRef} />
+                    </TabItem>
+                    <TabItem value="fromGroup" label="Aus Gruppe">
+                        <ImportFromGroupPopup {...props} popupRef={popupRef} />
+                    </TabItem>
+                    <TabItem value="fromList" label="Aus Liste">
+                        <ImportFromListPopup {...props} popupRef={popupRef} />
+                    </TabItem>
+                </Tabs>
```

Comment:
The Tabs are imo rendered a bit too prominent. Let's add a class to the tabs:

```tsx
<Tabs className={clsx(styles.tabs)}>
...
```

and then set the margin:

```scss
// styles.module.scss
.tabs {
    --ifm-tabs-padding-vertical: 0.15em;
    --ifm-tabs-padding-horizontal: 0.4em;
}
```